test/extended/router/gatewayapicontroller.go

```diff
@@ -246,6 +249,90 @@ var _ = g.Describe("[sig-network-edge][OCPFeatureGate:GatewayAPIController][Feat
 		g.By("Validating the http connectivity to the backend application")
 		assertHttpRouteConnection(defaultRoutename)
 	})
+
+	g.It("Ensure OSSM subscription, istiod deployment and the istio could be deleted and then get recreated [Serial]", func() {
+		const (
+			operatorNamespace      = "openshift-operators"
+			ingressNamespace       = "openshift-ingress"
+			ossmSubscriptionName   = "servicemeshoperator3"
+			csvName                = "servicemeshoperator3.v3.0.0"
+			ossmOperatorDeployment = "servicemesh-operator3"
+			istiodDeployment       = "istiod-openshift-gateway"
+			istioName              = "openshift-gateway"
+		)
```

Comment:
Some of those constants are defined in the upper level, so it's redundant to redefine them here. Also, it seems like most of the constants here can be used in many test cases, so they can also be moved upper.

Reply:
Thanks for the review. Yes, I think so, some common variables can be defined in the upper level. Maybe we can open another PR to update it. @lihongan @alebedev87 is it ok?

Reply:
I think we could update them in this PR.